apps/samradsgatt/components/ReviewCard/ReviewCard.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Text } from '@island.is/island-ui/core'
+import { format } from 'date-fns'
```

Comment:
lintið kvartar yfir þessu. Pæling að athuga hvort island-is sé með útfært einhverskonar format?